add example to support 8413 architecture (#35)
netmod-wg/schedule-yang: draft-ietf-netmod-schedule-yang.md

{{sec-mib}} discusses relationship with the managed objects defined in {{!RFC3231}}.

{{usage}} describes a set of examples to illustrate the use of the common schedule groupings ({{sec-grp}}).
And {{sec-ext}} provides sample modules to exemplify how future modules can use the extensibility
provisions in "ietf-schedule" ({{sec-schedule}}). Also, {{ex-framework}} provides

> examples of using "ietf-schedule" module for scheduled use of resources framework (e.g., {{?RFC8413}}).

Reviewer: ```suggestion
an example of using "ietf-schedule" module for scheduled use of resources framework (e.g., {{?RFC8413}}).
```

~~~~
{: #ex-13 title="Example of Scheduled Link's Bandwidth"}

> # Examples of Using "ietf-schedule" Module in scheduled based Framework

Reviewer: ```suggestion
# Examples of Using "ietf-schedule" Module for Scheduled Use of Resources Framework
```

   This section exemplifies how the architecture for supporting scheduled

>    reservation of TE resources in {{?RFC8413}} could leverage the "period-of-time"

Reviewer: ```suggestion
   reservation of Traffic Engineering (TE) resources in {{?RFC8413}} might leverage the "period-of-time"
```

   grouping defined in the "ietf-schedule" module to implement scheduled use of
   resources.

   The following example module shows how a scheduled link capacity reservation
   could be defined.

~~~~
{::include-fold ./yang/example-scheduled-capacity-reservation.yang}
~~~~

   {{Section 4 of ?RFC8413}} defines the reference architecture for scheduled use

>    of resources, the service requester sends a request to the PCE and includes the

Reviewer: ```suggestion
   of resources, the service requester sends a request to a Path Computation Element (PCE) and includes the
```

>    parameters of the LSP that the requester wishes to supply, the configuration

Reviewer: ```suggestion
   parameters of the Label Switched Path (LSP) that the requester wishes to supply, the configuration
```
